static/usr/lib/systemd/system-generators/rtc-sys-time-init-generator

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+# This generator instantiates rtc-sys-time-init@dev-rtc*.service by hooking it into 
+# sysinit.target when requested by kernel command line parameter rtc_sys_time_init=dev-rtc*.
+# Instances are based on the template rtc-sys-time-init@.service that is not generated.
+
+DEBUG_ENABLE="true"
+
+log() {
+    echo "$@" >> "/dev/kmsg"
+}
+
+debug() {
+    if [ "$DEBUG_ENABLE" = "true" ]; then
+        log "$@"
+    fi
+}
+
+normal_dir="$1"
+
+# Look for kernel command line parameter rtc_sys_time_init
+if ! dev_node=$(/usr/libexec/core/get-arg "rtc_sys_time_init"); then
+    debug "Kernel command line does not contain parameter rtc_sys_time_init"
+    exit 0
+fi
+
+# Allow value /dev/rtc and /dev/rtc0 to /dev/rtc9 (Assume no more than 9 RTCs)
+# TODO: Can the order of enumeration be guaranteed/controlled?
+if ! expr "$dev_node" : '^/dev/rtc[0-9]\{0,1\}$' >/dev/null; then
```

Comment:
I would not verify this in the generator. Otherwise it will go unnoticed if it does not work. Instead we want the service later to fail or timeout.

Reply:
Changed it to only issue warning, for extra visibility.